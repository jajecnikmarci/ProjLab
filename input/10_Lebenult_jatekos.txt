starttest
add room room1 3 poisonous
add room room2 3
add room room3 3
add door room1 room2 door1
add door room1 room3 door2
add item room1 Rug rug
add person room2 Student student1
add person room3 Professor professor1
gotoroom student1 room1
printstate room1
printstate student1
gotoroom student1 room2
pickup student1 rug
printstate room1
printstate student1
gotoroom professor1 room1
printstate room1
printstate student1
endtest