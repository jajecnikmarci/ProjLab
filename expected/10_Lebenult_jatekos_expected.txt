room1
	- capacity: 3
	- people: 
		student1
	- doors: 
	- items: 
	- poisonEffects: 
		poison1
	- stunEffects: 
	- lastCleaning: null
student1
	- location: room1
	- stunned: true
	- inventory: 
		rug
	- killImmunities: 
	- poisonImmunities: 
	- souls: 3
room1
	- capacity: 3
	- people: 
		student1
	- doors: 
	- items: 
	- poisonEffects: 
		poison1
	- stunEffects: 
	- lastCleaning: null
student1
	- location: room1
	- stunned: true
	- inventory: 
		rug
	- killImmunities: 
	- poisonImmunities: 
	- souls: 3
room1
	- capacity: 3
	- people: 
		student1
		professor1
	- doors: 
	- items: 
	- poisonEffects: 
		poison1
	- stunEffects: 
	- lastCleaning: null
student1
	- location: room1
	- stunned: true
	- inventory: 
		rug
	- killImmunities: 
	- poisonImmunities: 
	- souls: 2
